packages/oak/lib/core/Editable/Form.js

```diff
@@ -85,7 +85,7 @@ export default forwardRef(({
   ), [tabs, overrides]);
 
   const getFieldKeyTypes = useCallback(() => (
-    getFields().map(f => [f.key, f.type]).filter(f => f.[0])
```

Comment:
wtf

Reply:
it was useless, right. ?

Reply:
Either it's missing a `f?.[0]` or I'm just a dumbfuck. The later seems to be the case here.